multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -1468,6 +1468,16 @@ def find_class(self, module, name):
         renamed_module = module
         if module == "autogluon.text.automm.data.preprocess_dataframe":
             renamed_module = "autogluon.multimodal.data.preprocess_dataframe"
+        elif module == "autogluon.text.automm.data.process_categorical":
```

Comment:
This can be simplified as if module.startswith(“autogluon.text.automm”). We can then replace it with autogluon.multimodal